test.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+
+from DIRACbenchmark import *
+import pytest
+
+@pytest.mark.parametrize("copies, iterations, extraIteration", [
+	('single', 1, False),
+	('single', 2, True),
+	('wholenode', 1, False),
+	('wholenode', 2, True),
+	('jobslot', 1, False),
+	('jobslot', 2, True),
+	(2, 1, False),
+	(2, 2, True)
+])
+
+def testDIRACbenchmark(copies, iterations, extraIteration):
```

Comment:
This seems fine.
In theory, I think we should expect that:
1.  the result of two executions of the same function with the same input to be almost the same.
2. the result of two executions of the same function with a different number of iterations (1-2-5) to be almost the same too.
3. the result of two executions of different functions to be almost the same.

This seems to be the true for *1.* and *2.* but for some reason, `singleDiracBenchmark` and `wholenodeDiracBenchmark` seems to provide completely different results (and I guess it depends on the machine). 

I would add tests for *1.* and *2.*.
For *3.*, we have to figure out what happens (in the analysis part).